migrations.lock

```diff
@@ -158,6 +158,9 @@ blobs
  0011_blobmeta_compressed
  0012_rename_indexes
  0013_drop_icds_cas_index
+captcha
+ 0001_initial
+ 0002_delete_captchastore
```

Comment:
Not sure exactly how to handle this. It looks like it was a migration that wasn't added to the lock file. Shouldn't this change to the lock file have been made before Monday, the last shared commit this branch had with master?

Reply:
It's something to do with the captcha app being from installed requirements, not the HQ codebase. @joxl might have a more detailed explanation.

Reply:
Yes, this is one of the shortcomings of the migrations checker... if you have outdated Django plugins installed (which have migrations), they will show up when you render the `migrations.lock` file (happens automatically if you run `./manage.py makemigrations ...`).

@AddisonDunn, you can use `pip-sync` to update your installed python dependencies. It will remove the offending plugin. The exact command depends on which requirements `*.txt` file you use. For example, if you use `dev-requirements.txt`, you can perform this:

```bash
pip-sync requirements/dev-requirements.txt --dry-run  # optional
pip-sync requirements/dev-requirements.txt
```
This should remove the `django-simple-captcha` library, which is causing your extra migrations to show up. After that, rebuild and commit the corrected lock file:

```bash
./manage.py makemigrations --lock-update
```

Reply:
Thanks for the explanation and solution, Joel. Running `pip-sync` seems like it actually updated a few different modules.